tutoraspects/templates/aspects/apps/clickhouse/dev_config/server_config.xml

```diff
@@ -2,7 +2,7 @@
     {{patch("clickhouse-server-config")}}
 
     <logger>
-        <level>information</level>
+        <level>debug</level>
```

Comment:
I don't think this is what we want for production?

Reply:
isn't this the dev config?

Reply:
Oh duh, nevermind.